Use generic copyright holder name in license boilerplate

eparis opened the pull request:
The Google Contributor License Agreement (https://cla.developers.google.com/)
is not a copyright assignment. It is a license agreement. Thus it may be
incorrect for the author of a new file to claim that Google owned the
copyright in the header. Instead use a generic phrase "The Kubernetes Authors"
as the copyright holder.

eparis commented:
@smarterclayton

smarterclayton commented:
LGTM, would have @thockin or @brendandburns chime in for the Google side.  Discussed with the lawyers I assume?

eparis commented:
No. if someone on the google side wants a google lawyer to take a look, that sounds good to me.

lavalamp commented:
I don't think this is an accidental oversight. @sillsm for verification.

lavalamp commented:
@eparis @smarterclayton What do you think about just globally switching to "the Kubernetes Authors" instead of "Google, Inc"?

eparis commented:
Big fan. I know our legal is happy with that as well.  Assuming your side is good, I'll make the change.

lavalamp commented:
Yeah, SGTM (and our legal).

smarterclayton commented:
+1 - we sound all special and stuff

> On May 1, 2015, at 11:57 AM, Daniel Smith <notifications@github.com> wrote:
> 
> Yeah, SGTM (and our legal).
> 
> —
> Reply to this email directly or view it on GitHub.
>

thockin commented:
According to our legal folks, putting "Copyright Google" is not assigning
copyright, as much as it looks like that.  We can move to the other mode
"The Kubernetes Authors" but I think we thne have to keep an AUTHORS file
and make sure contributors are listed there.  Not sure if we should list
corp contribs (Red Hat Inc) or individuals.

On Fri, May 1, 2015 at 9:04 AM, Clayton Coleman <notifications@github.com>
wrote:

> +1 - we sound all special and stuff
>
> > On May 1, 2015, at 11:57 AM, Daniel Smith <notifications@github.com>
> wrote:
> >
> > Yeah, SGTM (and our legal).
> >
> > —
> > Reply to this email directly or view it on GitHub.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/7597#issuecomment-98166961>
> .
>

lavalamp commented:
The AUTHORS file was optional ("for vanity") if I understood the lawyers correctly.

eparis committed:
Make copyright ownership statement generic

Instead of saying "Google Inc." (which is not always correct) say "The
Kubernetes Authors", which is generic.

eparis commented:
updated.  Although how long any given rebase stays valid is an open question....

eparis commented:
rebased again, as people keep changing things   :)

lavalamp commented:
@jlowdermilk merging this to keep @eparis from spending the rest of his life rebasing-- hopefully it doesn't combine poorly with anything in flight.
